packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -322,12 +344,12 @@ const EAPostsItem = ({classes, ...props}: EAPostsItemProps) => {
             </div>
             <InteractionWrapper className={classes.interactionWrapper}>
               <PostsItemTrailingButtons
+                showArchiveButton={false}
```

Comment:
What's the reason for this change? (Not necessarily saying it's wrong)

Reply:
This is `true` by default, which meant that there was an archive icon that appeared to the right of the post item on hover if the post was a draft. Clicking on the icon archives the draft post. You can see this if you view a user's "My Drafts" list in their profile. We don't have good support for letting users organize their drafts in general (ex. we don't have the sorting/filtering options that LW does), and I would guess that archiving your draft on the EA Forum is basically a black hole, so I figured we should just hide that icon for now.